gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc

```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
+. Click on the *Flex Gateways* tab in the side navigation panel.
+. Click *Add Gateway*.
+. Select *Docker*.
+. Copy the code block in the *Register your gateway* section. The code block will contain your unique organization ID and token.
++
+The following screenshot of the *Add a Flex Gateway* page highlights the code block that should be copied.
++
+image::gateway-quickstart-add-gateway.png["Screenshot of 'Add a Flex Gateway'",width=90%]
+
+. Run the copied code block in your terminal window, making sure to replace `<gateway-name>` with a name, such as `my-gw`.
++
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+. Select *Flex Gateways* in the side navigation panel.
++
+Your registered Flex Gateway now appears in the list of gateways.
+
+== Run Flex Gateway
+
+To run Flex Gateway, run the following command:
+
+[source,ssh]
+----
+docker run --rm \
+  -v "$(pwd)":/usr/local/share/mulesoft/flex-gateway/conf.d \
+  -p 8081:8081 \
+  mulesoft/flex-gateway
+----
+
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+A new Flex Gateway instance is now running in your terminal window.
+
+== Publish a Simple API
+
+. Select *API Manager* from the Anypoint Platform menu.
+. Click *Add API*, followed by *Add new API*.
++
+image::gateway-quickstart-add-new-api.png["Screenshot of 'Add new API'",width=35%]
+
+. In the *Add API* page, select *Flex Gateway* as your runtime.
+. Select your registered gateway in the *Select a gateway* section.
+. Click *Next*.
++
+The following screenshot of the *Add API* page highlights the specified options.
++
+image::gateway-quickstart-select-gateway-for-api.png["Screenshot of 'Select Gateway for API'",width=100%]
+
+. Click *Create New API*.
+. Specify an API name and select *HTTP API* as the asset type.
+. Click *Next*.
+. In the *Endpoint* page, enter the following for *Implementation URI*:
++
+[source]
+----
+https://jsonplaceholder.typicode.com/
+----
+. For *Port*, enter `8081`.
+. Click *Next*.
++
+// The following screenshot of the *Endpoint* page highlights the specified options.
+// +
+// image::gateway-quickstart-select-endpoint.png["Screenshot of 'Endpoint' page",width=100%]
+
+. Click *Save & Deploy*.
++
+Inside the gateway instance Docker container, all HTTP requests made to `localhost` on port `8081` will now be proxied to the `jsonplaceholder` service.
+. To test the API Instance, run the following command in a new terminal window:
++
+[source,ssh]
+----
+curl -s -o /dev/null -w "%{http_code}" --request GET 'http://localhost:8081/'
+----
++
+The command executes a `GET` request to the API Instance, and subsequently prints the resulting `200` status code, indicating success.
+
+== Secure Your API Using the Basic Authentication Policy
+
+. While still in *API Manager*, select *API Administration* in the side navigation panel.
+. Select the highlighted name of the API Instance created in the previous section.
```

Comment:
hmmm. Maybe Instance is a proper noun?

Reply:
I removed "instance", as it was redundant.